examples/eager_examples/scripts/example_reset.py

```diff
@@ -6,7 +6,8 @@
 from eager_core.eager_env import EagerEnv
 from eager_core.objects import Object
 from eager_core.wrappers.flatten import Flatten
-from eager_bridge_webots.webots_engine import WebotsEngine
+from eager_bridge_webots.webots_engine import WebotsEngine  # noqa: F401
```

Comment:
This is for the PEP8 testing, otherwise it would fail because something is imported but not used. This make sure that that specific warning is ignored for this line.